client/src/components/list/ReviewCardLists.jsx

```
@@ -1,7 +1,14 @@
+import styles from './ReviewCardLists.module.css';
+import ReviewCard from '../card/ReviewCard';
 
-export default function ReviewCardLists() {
+export default function ReviewCardLists({ reviewCardLists  }) {
   return (
-    <div>ReviewCardLists</div>
-  )
+    <div className={styles.wrapper}>
+      {reviewCardLists?.map((reviewDetail) => {
+        const { review, book, user } = reviewDetail;
+        return <ReviewCard review={review} book={book} user={user}></ReviewCard>;
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)